create-artists/README.md

```diff
@@ -0,0 +1,6 @@
+### Initial Setup
+1. Install pyenv
+2. Run `pyenv install` to install Python version
+2. Install (poetry)[https://python-poetry.org/]
+3. Run `poetry install` to install packages
+4. Run `poetry run python main.py` to download all images
```

Comment:
`Download all images` is a holdover from copy/paste (I assume).

I'm also having a bit of trouble with the install steps, but that's not your fault, as you copied my previous scripts, so I think I've found an issue in my other scripts. Let me figure that out, and I'll get back to you on what to change.

Thank you for the PR, by the way!

Reply:
Alright, *finally* had a chance to figure this out - apologies for the delay!

1. The `create-artists.py` file should be changed to `main.py` to matchup with the README instructions and the other sub-projects in this repo.
2. `download all images` in the last step should be updated to reflect what the project is actually doing
3. Fix the numbering (also a holdover of my README mistake, sorry!)
4. The README should be updated to match the newly updated READMEs (namely clarifying that the last two steps should be run with `pyenv exec`